images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -11,12 +11,19 @@ function Get-OSName {
 function Get-OSVersion {
     $OSVersion = (Get-CimInstance -ClassName Win32_OperatingSystem).Version
     $OSBuild = (Get-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion' UBR).UBR
-    return "OS Version: $OSVersion Build $OSBuild"
+    return "$OSVersion Build $OSBuild"
+}
+
+function Build-OSInfoSection {
+    $osInfoNode = [HeaderNode]::new($(Get-OSName))
+    $osInfoNode.AddToolVersion("OS Version:", ($(Get-OSVersion)))
```

Comment:
nit: remove extra brackets

Reply:
Oops, sorry. I think it can be removed only in the second line. Not in both.

Reply:
ok, removed